packages/opentelemetry-sdk-trace-base/src/Span.ts

```diff
@@ -64,6 +64,7 @@ export class Span implements APISpan, ReadableSpan {
   private readonly _spanContext: SpanContext;
   readonly kind: SpanKind;
   readonly parentSpanId?: string;
```

Comment:
let's mark this as `@deprecated` then if we're going to do a feature-release anyway.